contracts/interfaces/draft-IERC7579.sol

```diff
@@ -93,6 +105,11 @@ struct Execution {
     bytes callData;
 }
 
+/**
+ * @dev Smart Account execution according to ERC-7579.
+ *
+ * To implement ERC-7579 modules, smart accounts must implement this interface.
```

Comment:
```suggestion
 * @dev ERC-7579 Execution.
 *
 * Accounts should implement this interface so that the Entrypoint and ERC-7579 modules can execute operations.
```